config/crd/bases/apps.foundationdb.org_foundationdbclusters.yaml

```diff
@@ -7604,7 +7604,7 @@ spec:
                 updatePodsByReplacement:
                   type: boolean
                 version:
-                  pattern: ^(\d+)\.(\d+)\.(\d+)$
+                  pattern: (\d+)\.(\d+)\.(\d+)
```

Comment:
We may want to run/add some tests to ensure versions like `prerelease-5.6.2` and `test-1.2.3-test` etc. are actually working like expected.

Reply:
That's fine to only test the `ParseFdbVersion` the Kubernetes validation should behave the same way.